src/dotnet/Chat.Service/Controllers/ChatsController.cs

```diff
@@ -96,6 +96,10 @@ public Task<bool> CanSendPeerChatMessage(Session session, string chatPrincipalId
     public Task<ImmutableArray<Author>> ListMentionableAuthors(Session session, string chatId, CancellationToken cancellationToken)
         => _service.ListMentionableAuthors(session, chatId, cancellationToken);
 
+    [HttpGet, Publish]
+     public Task<ImmutableArray<ReactionSummary>> ListMessageReactions(Session session, Symbol chatId, long entryId, CancellationToken cancellationToken)
```

Comment:
Can we actually move all of reaction-related stuff to dedicated services?

IMO `Chats` is already polluted well enough...

Reply:
+ In this case we could use much simpler names - `Reactions.GetSummary / List`, etc.

Reply:
done